tasks/main.yml

```diff
@@ -52,6 +52,7 @@
         name: libselinux-python
         state: present
       when: ansible_os_family == "RedHat" and ansible_distribution_major_version >= 7
+      ignore_errors: true # The when statement above is not reliable on hLinux
```

Comment:
`when: ansible_fqdn is undefined` is a poor test of "has fact checking run on this host yet".

I don't see any alternative other than just always running fact checking. Running `setup` with every deploy will slow this already slow role down even more but it's probably better to be slow and correct, than fast and broken.

I'll update the PR accordingly and you can judge for yourself.